api/v1alpha1/clustergroupupgrade_types.go

```diff
@@ -61,6 +61,9 @@ type ClusterGroupUpgradeSpec struct {
 	// Important: Run "make" to regenerate code after modifying this file
 	Clusters            []string                 `json:"clusters,omitempty"`
 	RemediationStrategy *RemediationStrategySpec `json:"remediationStrategy,omitempty"`
+	ManagedPolicy       []string                 `json:"managedPolicy,omitempty"`
```

Comment:
Since this field is a list should the field name be plural? ie 
```suggestion
	ManagedPolicies     []string                 `json:"managedPolicies,omitempty"`
```